React/Views/RCTHandledKey.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#if TARGET_OS_OSX // [macOS
+#import <React/RCTConvert.h>
+
+@interface RCTHandledKey : NSObject
```

Comment:
Could we add a comment for this class as well? Just to distinguish this from other key event objects, and to make it more obvious this is used for filtering